src/communication.rs

```diff
@@ -69,6 +70,18 @@ impl PartialEq for Sender {
     }
 }
 
+// Fixes #255.
```

Comment:
Can you remove both of these comments ? I don't think they bring anything to the code that isn't already in the git history. Also if the repo ever moves or anything happens to github, the references would be lost anyway

Reply:
Removed in the latest commit.